### DISCUSSION TITLE
Modificacion struct Head

### DISCUSSION THREAD

**Discussion by SCarcG1** [IIC2133-PUC/2024-1#117] [Tarea 1] [answered]
Hola,
tengo una duda con respecto a si se pueden agregar atributos extra al struct Head, dado a que estaba buscando diferentes formas de realizar el factor de orden de llegada, y pensaba agregar un atributo al respecto para facilitarlo.
La pregunta va mas que nada por que Head fue entregado e inicializado en el codigo provisto, por lo que tendria que modificar este codigo generado al forma el repo

**Comment by PabloNavarroY** [comment, marked as answer]
Sí puedes
